README.md

```diff
@@ -267,6 +268,45 @@ Prints this:
 {image24x36}{noBreakSpace}Monday{enDash}Friday
 ```
 
+## Tag Styles
+
+BonMot can style text between arbirtrary tags using a `<tag></tag>` format and `\` as an escape character.
```

Comment:
#nth it would be nice to call out the (or at least, my) intended use of this feature: to parse localized strings.